CrossSectionAnalysis/CrossSectionAnalysis.py

```diff
@@ -588,6 +613,20 @@ def selectCurveInMarkupsModule(self):
     else:
         markupsWidget.setEditedNode(None)
 
+# Convenience function to get the point of minimum or maximum diameter. Is useful for arterial stenosis (minimum) or aneurysm (maximum).
+  def getExtremeDiameterPoint(self, boolMaximum):
+    if self.inputSliceNode is None or self.inputPath is None or (self.vmtkCenterlineRadii.size == 0):
+        return -1
+    target = -1
+    for i in range(self.vmtkCenterlineRadii.size):
```

Comment:
You can get index of max/min value using numpy - see https://thispointer.com/find-max-value-its-index-in-numpy-array-numpy-amax/. It would much faster and less code.